_data/sidebar.yml

```diff
@@ -126,6 +126,8 @@ docs:
         url: /documentation/bundle-plugin.html
       - page: Versioning in the Container
         url: /documentation/jdisc/component-versioning.html
+      - page: Using ZooKeeper
+        url: /documentation/using-zookeeper.html
```

Comment:
@kkraune: Should this be `.md` in this case?

Reply:
markdown files are auto-converted to html, so using .html in the link is correct